src/components/ebay-date-textbox/date-textbox.stories.js

```diff
@@ -0,0 +1,127 @@
+import Readme from './README.md';
+import Component from './examples/default.marko';
+import code from './examples/default.marko?raw';
```

Comment:
Id like this to be named
DefaultTemplateCode
And the template should be DefaultTemplate. That should be the new standard that we're using